lib/easypost/address.rb

```diff
@@ -6,24 +6,19 @@ class EasyPost::Address < EasyPost::Resource
 
   # Create an Address.
   def self.create(params = {}, api_key = nil)
-    url = self.url
-
     address = params.reject { |k, _| [:verify, :verify_strict].include?(k) }
 
-    if params[:verify] || params[:verify_strict]
-      verify = params[:verify] || []
-      verify_strict = params[:verify_strict] || []
-
-      url += '?'
-      verify.each do |verification|
-        url += "verify[]=#{verification}&"
-      end
-      verify_strict.each do |verification|
-        url += "verify_strict[]=#{verification}&"
-      end
+    wrapped_param = { address: address }
```

Comment:
Change the name of `wrapped_param` to `wrapped_params` and we should be set.

Reply:
Fixed